app/components/meetups/mainstage/Mainstage.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import { Collapse } from 'react-bootstrap';
+import styles from '../../../styles/meetup.module.css';
+import dynamic from 'next/dynamic';
+import { useMediaQuery } from '@rocket.chat/fuselage-hooks';
```

Comment:
With the latest changes, we are no longer using the `fuselage-hooks`, so please use an alternative maybe available in the `react-bootstrap` to catch the screen size breakpoints.

Reply:
Sure thing, I've made the changes and am now using a custom hook to calculate breakpoints